bin/openupgrade/openupgrade_log.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-from openupgrade_tools import table_exists
+from openupgradelib import openupgrade_tools
```

Comment:
openupgrade_tools is present in bin/openupgrade this is needed to make sure the openupgrade_tools from openupgradelib is chosen.